staging/src/k8s.io/mount-utils/mount_helper_common.go

```diff
@@ -122,6 +122,10 @@ func removePathIfNotMountPoint(mountPath string, mounter Interface, extensiveMou
 	}
 
 	if err != nil {
+		if os.IsNotExist(err) {
+			klog.V(4).Infof("%q does not exist", mountPath)
+			return notMnt, nil
```

Comment:
Currently it seems to be the case that notMnt=true in this case. But maybe we want to make that explicit and just return true, nil here instead of depending on the implementation of Is*NotMountPoint? I think that might make behavior more predicatble.

Reply:
I updated it to true, thanks!

Reply:
Is it possible to add a unit test for this?